content/applications/marketing/marketing_automation/target_audience.rst

```diff
@@ -1,123 +1,123 @@
-:guilabel:`Opportunity`. The number of :guilabel:`Records` that fit this specific filter equation
-changes as the equation is customized.
-
-Add another node to this filter by clicking the :guilabel:`➕ (plus sign)` icon to the right of the
-equation.
-
-With "new" opportunities being the target of this filter, the second node will focus on *only*
-locating opportunities that are in the :guilabel:`New` stage of the pipeline. To do that, select
-the default :guilabel:`ID` from the first portion of the second filter equation, and scroll down (or
-search for) :guilabel:`Stage` from the field drop-down menu.
-
-Once again, leave the second portion of the filter equation on :guilabel:`🟰 (equal sign)` icon.
-
-Lastly, highlight the default value in the third (and final) portion of the second filter equation,
-and type in `New`. With that in place, Odoo only targets opportunities that are in the "New" stage
-of the pipeline.
-
-.. image:: target_audience/filters-opportunities.png
-   :align: center
-   :alt: A standard scenario using filters in the Odoo Marketing Automation app.
-
-Scenario #2: Narrow down target to event attendees who purchased a specific ticket
-----------------------------------------------------------------------------------
-
-While in *Edit mode* on a campaign template form (by clicking the :guilabel:`Edit` button), select
-the :guilabel:`Target` field, and click :guilabel:`Search More` from the drop-down menu. Then,
-scroll down (or search for) :guilabel:`Event`, and select it.
-
-Next, click :guilabel:`Add Filter` in the :guilabel:`Filter` field. Click on the default
-:guilabel:`ID` filter option in the first portion of the filter equation. Doing so reveals a
-drop-down menu full of filter options. From this drop-down, scroll down (or search for)
-:guilabel:`Event`.
-
-Click the default :guilabel:`🟰 (equal sign)` icon in the second portion of the filter equation. This
-reveals a drop-down menu. From this drop-down menu, select :guilabel:`contains`.
+- :guilabel:`Target`: defines the model of the target audience for the campaign. The assigned model
+  determines the fields that are available throughout the campaign, including the fields available
+  in the :guilabel:`Filter` section, and in dynamic placeholders.
+- :guilabel:`Save as Favorite Filter`: saves the current :guilabel:`Filter` for future use with the
+  current :guilabel:`Target` model, and can be managed from the :menuselection:`Marketing Automation
+  app --> Configuration --> Favorite Filters` menu.
+- :guilabel:`Unicity based on`: specifies the :guilabel:`Target` model field at which duplicates
+  should be avoided. Traditionally, the :guilabel:`Email` field is used, but any available field can
+  be used.
+- :guilabel:`Filter`: contains an interactive form to refine the target audience on a specific
+  subset of the :guilabel:`Target` model, detailed in the
+  :ref:`marketing_automation/defining-filters` section.
```

Comment:
"Specific subset of the  **Target** model" is likely going to confuse non-technical people so here's a suggested rewrite:

```suggestion
- :guilabel:`Filter`: contains an interactive form with configurable logic to further refine the
  targeting parameters under the chosen :guilabel:`Target` model. See more details in the
  :ref:`marketing_automation/defining-filters` section.
```